This patch is safe, because all Debian systems should be POSIX compliant
enough to have <unistd.h> and related headers, whether or not they have
'unix' as a compiler symbol (NetBSD, in particular, lacks it).

diff -ur zlib-1.1.4.orig/contrib/minizip/Makefile zlib-1.1.4/contrib/minizip/Makefile
--- zlib-1.1.4.orig/contrib/minizip/Makefile	2002-10-10 04:24:24.000000000 +0000
+++ zlib-1.1.4/contrib/minizip/Makefile	2002-10-10 04:25:09.000000000 +0000
@@ -1,5 +1,5 @@
 CC=cc
-CFLAGS=-O2 -g -I../..
+CFLAGS=-O2 -g -I../.. -Dunix
 
 UNZ_OBJS = miniunz.o unzip.o
 ZIP_OBJS = minizip.o zip.o
